tests/qos/test_qos_sai.py

```diff
@@ -233,6 +233,11 @@ def testQosSaiPfcXoffLimit(
         if 'cell_size' in qosConfig[xoffProfile].keys():
             testParams["cell_size"] = qosConfig[xoffProfile]["cell_size"]
 
+        duthost = dutConfig['dutInstance']
+        dutTestParams['basicParams']["router_mac"] = duthost.get_dut_iface_mac(
```

Comment:
Will this change affect single asic platforms?

Reply:
I verified that the single asic DUT also can work with this code. In my run I encountered a different problem, but the router mac is definetly populated in the PTF command line.